.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.5']
+        django-version: ['django22']
+        toxenv: ['django22', 'quality', 'docs', 'pii-annotations']
```

Comment:
I think you should add jasmine tests too in the tox environments and mark them as allow_failures.

Reply:
Added it as a ticket after talking with George about doing this as follow up work. Would rather not have things running that have no intention of passing. Can be tracked at https://openedx.atlassian.net/browse/ENT-3880